src/server.c

```diff
@@ -6029,7 +6040,9 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "aof_last_write_status:%s\r\n", (server.aof_last_write_status == C_OK && aof_bio_fsync_status == C_OK) ? "ok" : "err",
                 "aof_last_cow_size:%zu\r\n", server.stat_aof_cow_bytes,
                 "module_fork_in_progress:%d\r\n", server.child_type == CHILD_TYPE_MODULE,
-                "module_fork_last_cow_size:%zu\r\n", server.stat_module_cow_bytes));
+                "module_fork_last_cow_size:%zu\r\n", server.stat_module_cow_bytes,
+                "slot_migration_fork_in_progress:%d\r\n", server.child_type == CHILD_TYPE_SLOT_MIGRATION,
+                "slot_migration_fork_last_cow_size:%zu\r\n", server.stat_slot_migration_cow_bytes));
```

Comment:
```suggestion
                "slot_migration_fork_in_progress:%d\r\n", server.child_type == CHILD_TYPE_SLOT_MIGRATION));
```
I also think we should get rid of the global COW size, since the amount of time spent for each slot migration will change, I'm not sure having how much COW memory is useful. I think we should add it to the `GETSLOTMIGRATIONS` command instead.